boards/arm/adafruit_trinket_m0/pre_dt_board.cmake

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2021 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+# SPI is implemented via sercom so node name isn't spi@...
+list(APPEND EXTRA_DTC_FLAGS "-Wno-spi_bus_bridge")
```

Comment:
Do we need to do this for the flexcomm spi on lpcxpresso and mimxrt685 boards too?

Reply:
No, I'm guessing they covered by the fact we set "-Wno-unique_unit_address" globally in camek/dts.cmake covers them.